src/app/admin/admin-issuer-edit-campaign-visibility/admin-issuer-edit-campaign-visibility.component.html

```diff
@@ -0,0 +1,53 @@
+<ng-container *ngIf="(issuer$ | async) as issuer">
+  <ng-container *ngIf="(offers$ | async) as offersRes">
+    <ng-container *ngIf="(offersDisplaySettings$ | async) as offersDisplaySettingsRes">
+      <ng-container *ngIf="offersRes.value && offersDisplaySettingsRes.value">
+        <div class="flex flex-col items-center px-2 justify-center w-full">
+          <div class="flex flex-col lg:flex-row gap-4 my-4">
+            <button app-action-button
+                    class="flex order-2 lg:order-1 items-center justify-center gap-4 bg-indigo-800 text-white text-lg
+                           px-24 py-2 rounded-full transition-all ease-in-out duration-300"
+                    text="Discard changes"
+                    [onClick]="discardChanges.bind(this)">
+            </button>
```

Comment:
"Discard changes" is really misleading for me. With this button you don't discard anything, you are going back to the issuer screen. Discard changes button would be one that reset your changes to the default, pre-edit state. I don't think we need it here, as you can always go back like on any other screen.